<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Color</class>
 <widget class="QDockWidget" name="Color">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>163</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>230</width>
    <height>163</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 9pt &quot;Microsoft YaHei UI&quot;</string>
  </property>
  <property name="windowTitle">
   <string>Color</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true">font: 8pt &quot;Microsoft YaHei UI&quot;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QComboBox" name="cbox_type">
         <item>
          <property name="text">
           <string>PointCloud</string>
          </property>
          <property name="icon">
           <iconset resource="../resources/res.qrc">
            <normaloff>:/res/icon/cloud_type.svg</normaloff>:/res/icon/cloud_type.svg</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>BoundingBox</string>
          </property>
          <property name="icon">
           <iconset resource="../resources/res.qrc">
            <normaloff>:/res/icon/boundingbox.svg</normaloff>:/res/icon/boundingbox.svg</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Background</string>
          </property>
          <property name="icon">
           <iconset resource="../resources/res.qrc">
            <normaloff>:/res/icon/background.svg</normaloff>:/res/icon/background.svg</iconset>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout"/>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="btn_apply">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Apply</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/res.qrc">
             <normaloff>:/res/icon/apply.svg</normaloff>:/res/icon/apply.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_reset">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Reset</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/res.qrc">
             <normaloff>:/res/icon/reset.svg</normaloff>:/res/icon/reset.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
